src/libraries/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableSortedSet.cs

```diff
@@ -123,7 +123,7 @@ public static ImmutableSortedSet<T> Create<T>(IComparer<T>? comparer, params T[]
         /// <param name="comparer">The comparer.</param>
         /// <param name="items">The items to prepopulate.</param>
         /// <returns>The new immutable collection.</returns>
-        public static ImmutableSortedSet<T> Create<T>(IComparer<T>? comparer, ReadOnlySpan<T> items)
```

Comment:
I'm glad this was caught by apicompat in CI but I was expecting that my local build would've caught it.
cc @dotnet/area-infrastructure-libraries